tools/src/test/resources/log4j-off.properties

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Do we really need a custom log4j file? Or could we reuse the existing `tools-log4j.properties` file?

Reply:
+1

Reply:
Without custom logging tool output overwhelmed with the logs like the following.
It almost impossible to find actual tool log indide trace messages.
Do we really want to have it?
Huge logging affect tool performance, also

```
2023-11-08 20:26:03,643] TRACE [KafkaApi-101] Handling request:RequestHeader(apiKey=PRODUCE, apiVersion=10, clientId=producer-1, correlationId=4876, headerVersion=2) -- {acks=0,timeout=20000,partitionSizes=[my-topic-7=100072]} from connection 127.0.0.1:60353-127.0.0.1:60370-1;securityProtocol:PLAINTEXT,principal:User:ANONYMOUS (kafka.server.KafkaApis:54)
[2023-11-08 20:26:03,643] DEBUG Completed request:{"isForwarded":false,"requestHeader":{"requestApiKey":0,"requestApiVersion":10,"correlationId":4886,"clientId":"producer-1","requestApiKeyName":"PRODUCE"},"request":{"transactionalId":null,"acks":0,"timeoutMs":20000,"topicData":[{"name":"my-topic","partitionData":[{"index":3,"recordsSizeInBytes":100072}]}]},"response":"","connection":"127.0.0.1:60356-127.0.0.1:60372-0","totalTimeMs":4.409,"requestQueueTimeMs":0.144,"localTimeMs":4.209,"remoteTimeMs":0.0,"throttleTimeMs":0,"responseQueueTimeMs":0.055,"sendTimeMs":0.0,"securityProtocol":"PLAINTEXT","principal":"User:ANONYMOUS","listener":"PLAINTEXT","clientInformation":{"softwareName":"apache-kafka-java","softwareVersion":"3.7.0-SNAPSHOT"}} (kafka.request.logger:284)
[2023-11-08 20:26:03,643] TRACE Socket server received empty response to send, registering for read: Response(type=NoOp, request=Request(processor=1, connectionId=127.0.0.1:60356-127.0.0.1:60372-0, session=Session(User:ANONYMOUS,/127.0.0.1), listenerName=ListenerName(PLAINTEXT), securityProtocol=PLAINTEXT, buffer=null, envelope=None)) (kafka.network.Processor:54)
[2023-11-08 20:26:03,642] DEBUG [ReplicaManager broker=104] Produce to local log in 3 ms (kafka.server.ReplicaManager:62)
[2023-11-08 20:26:03,643] TRACE Processor 1 received request: RequestHeader(apiKey=PRODUCE, apiVersion=10, clientId=producer-1, correlationId=4893, headerVersion=2) -- {acks=0,timeout=20000,partitionSizes=[my-topic-8=100072]} (kafka.network.RequestChannel$:47)
[2023-11-08 20:26:03,642] TRACE [UnifiedLog partition=my-topic-14, dir=/Users/user/src/kafka/data/kafka-7554232574969976698] Appended message set with last offset: 340, first offset: 340, next offset: 341, and messages: MemoryRecords(size=100072, buffer=java.nio.HeapByteBuffer[pos=0 lim=100072 cap=100075]) (kafka.log.UnifiedLog:54)
[2023-11-08 20:26:03,642] TRACE [KafkaApi-100] Handling request:RequestHeader(apiKey=PRODUCE, apiVersion=10, clientId=producer-1, correlationId=4885, headerVersion=2) -- {acks=0,timeout=20000,partitionSizes=[my-topic-1=100072]} from connection 127.0.0.1:60350-127.0.0.1:60369-0;securityProtocol:PLAINTEXT,principal:User:ANONYMOUS (kafka.server.KafkaApis:54)
[2023-11-08 20:26:03,643] TRACE [UnifiedLog partition=my-topic-14, dir=/Users/user/src/kafka/data/kafka-7554232574969976698] Setting high watermark (offset=341, segment=[0:34124552]) (kafka.log.UnifiedLog:54)
[2023-11-08 20:26:03,643] DEBUG [Partition my-topic-14 broker=103] High watermark updated from (offset=340, segment=[0:34024480]) to (offset=341, segment=[0:34124552]) (kafka.cluster.Partition:62)
[2023-11-08 20:26:03,643] TRACE [ReplicaManager broker=103] 100072 written to log my-topic-14 beginning at offset 340 and ending at offset 340 (kafka.server.ReplicaManager:54)
[2023-11-08 20:26:03,643] TRACE [Kafka Request Handler 4 on Broker 102], Kafka request handler 4 on broker 102 handling request Request(processor=1, connectionId=127.0.0.1:60356-127.0.0.1:60372-0, session=Session(User:ANONYMOUS,/127.0.0.1), listenerName=ListenerName(PLAINTEXT), securityProtocol=PLAINTEXT, buffer=java.nio.HeapByteBuffer[pos=100121 lim=100121 cap=100121], envelope=None) (kafka.server.KafkaRequestHandler:54)
```

Reply:
I removed `log4j-off.properties` file and use `log4j.properties` directly from core module.
Alternatively we can configure logging by hardcoded properties in tool initializer. Like the following
Please, write me what is the best way to go.

```
        Properties logProps = new Properties();

        logProps.put("log4j.rootLogger", "OFF, stdout");
        logProps.put("log4j.appender.stdout", "org.apache.log4j.ConsoleAppender");
        logProps.put("log4j.appender.stdout.layout", "org.apache.log4j.PatternLayout");
        logProps.put("log4j.appender.stdout.layout.ConversionPattern", "[%d] %p %m (%c:%L)%n");
        logProps.put("log4j.logger.kafka", "WARN");
        logProps.put("log4j.logger.org.apache.kafka", "WARN");
```